pttrack/templates/pttrack/workup_detail.html

```diff
@@ -22,7 +22,9 @@
     <div class="row text-center">
         <h4>Actions</h4>
         <div class="btn-group">
+            {% if workup.signed and request.session.clintype_pk == "Attending" or not workup.signed %}
```

Comment:
I get what you are saying, but I am not sure if that will work. I'll look into a work around